src/main/java/com/google/step/servlets/PromoCodeServlet.java

```diff
@@ -1,5 +1,18 @@
 package com.google.step.servlets;
 
-public class PromoCodeServlet {
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
+@WebServlet("/promo-code")
+public class PromoCodeServlet extends HttpServlet {
+
```

Comment:
I believe we need an init function

Reply:
I don't think that's the case with servlets.

Reply:
It's not needed for servlets, but we want it to initialize the youtubescraper.